microsoft-edge/webview2/reference/javascript/webview.yml

```
@@ -0,0 +1,386 @@
+### YamlMime:TSType
+name: WebView
+uid: 'WebView2Script!WebView:class'
+package: WebView2Script!
+fullName: WebView
+summary: >-
+  `window.chrome.webview` is the class to access the WebView2-specific APIs that are available to the script running
+  within WebView2 Runtime.
+remarks: ''
+isPreview: false
+isDeprecated: false
+type: class
+properties:
+  - name: hostObjects
+    uid: 'WebView2Script!WebView#hostObjects:member'
+    package: WebView2Script!
+    fullName: hostObjects
+    summary: >-
+      Contains asynchronous proxies for all host objects added via `CoreWebView2.AddHostObjectToScript` as well as
+      options to configure those proxies, and the container for synchronous proxies.
+
+
+      If you call `coreWebView2.AddHostObjectToScript("myObject", object);` in your host code then an asynchronous proxy
+      for `object` will be exposed in script as `chrome.webview.hostObjects.myObject`<!-- -->.
+    remarks: ''
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: 'hostObjects: HostObjectsAsyncRoot;'
+      return:
+        type: '<xref uid="WebView2Script!HostObjectsAsyncRoot:class" />'
+        description: >-
+
+
+          #### Examples
+
+
+
+          For example, suppose you have a COM object with the following interface.

```

```suggestion
          For example, suppose you have a COM object with the following interface:
```
i favor colons when possible, to glue flow